### DISCUSSION TITLE
For VolView hacking, a script starting a HTTP server

### DISCUSSION THREAD

**Discussion by PaulHax** [Kitware/VolView#498] [Show and tell]
Pushed up a repo I sometimes use when iterating on a feature/fix.  Running `./start.sh` in there starts a HTTP server and opens VolView in Chrome  with URL parameters to load a config.json/session.volview.zip/small-dicom.  Saves a few clicks.

https://github.com/PaulHax/volview-backend

**Comment by floryst** [comment]
Nicely done!
